src/workerd/io/compatibility-date.c++

```diff
@@ -356,4 +356,11 @@ kj::Maybe<PythonSnapshotRelease::Reader> getPythonSnapshotRelease(
   return result;
 }
 
+kj::String getPythonBundleName(PythonSnapshotRelease::Reader pyodideRelease) {
```

Comment:
Not sure if this code fits logically within compatibility-date.c++ does it? I guess we have a lot of other pyodideRelease code here already but still